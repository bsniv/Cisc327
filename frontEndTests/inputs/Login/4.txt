login
agent
createacct 1234567 jeff
logout