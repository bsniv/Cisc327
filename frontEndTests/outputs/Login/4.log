Please login:
Enter 'machine' for a normal session, or 'agent' for an agent session:
Enter 'machine' for a normal session, or 'agent' for an agent session:
Enter 'machine' for a normal session, or 'agent' for an agent session:
Enter 'machine' for a normal session, or 'agent' for an agent session:
