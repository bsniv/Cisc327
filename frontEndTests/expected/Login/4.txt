NEW 1234567 000 0000000 jeff
EOS 0000000 000 0000000 ***